wpengine/atlas-blueprint-portfolio
# Merl 711 update blueprints to use 2023 default theme

[MERL-711](https://wpengine.atlassian.net/jira/software/c/projects/MERL/boards/1007?modal=detail&selectedIssue=MERL-711) changes the theme mods and current theme to 2023 in the acm.json file as an update for Atlas Portal.

**To test:** 
Clone this branch and deploy it in [Atlas Portal. ](https://my.wpengine.com/atlas#/)
Check the resulting layout on the resulting site to ensure the update does not introduce any layout shifts or breaks.

[MERL-711]: https://wpengine.atlassian.net/browse/MERL-711?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
